guides/source/association_basics.md

```diff
@@ -1305,6 +1305,20 @@ The `:source` option specifies the source association name for a `has_one :throu
 
 The `:source_type` option specifies the source association type for a `has_one :through` association that proceeds through a polymorphic association.
 
+```ruby
+class Hdhp < ApplicationRecord; end
+class Ppo < ApplicationRecord; end
+
+class Employment < ApplicationRecord
+  has_one :health_care_plan, polymorphic: true
+end
+
+class User < ApplicationRecord
+  has_one :employment
+  has_one :hdhp, through: :employment, source: :health_care_plan, source_type: "Hdhp"
+end
+```
```

Comment:
Can you use a different example? I think that health care plan is confusing for non-US readers. Even US folks may not understand the difference between PPO and high deductible plan. I have one and had to look up "HDHP". :smile:

Reply:
@eileencodes I'd be really happy to. My first thought was to extend one of the existing example domains, but I had the hardest time coming up with a polymorphic has_one :through example that made sense. I totally agree that the one I settled on is not very obvious/accessible. I'll give this some more thought, though I'd happily take a suggestion!